src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportHelper.java

```diff
@@ -104,28 +102,57 @@ public static ListingData processClaimsAcceptedRequest(ListingDetails listingDet
 
     public static ListingData processLiveCaseloadRequest(ListingDetails listingDetails,
                                                          List<SubmitEvent> submitEvents) {
-
         if (submitEvents != null && !submitEvents.isEmpty()) {
             log.info(CASES_SEARCHED + submitEvents.size());
             List<AdhocReportTypeItem> localReportsDetailList = new ArrayList<>();
+
             for (SubmitEvent submitEvent : submitEvents) {
                 AdhocReportTypeItem localReportsDetailItem =
                         getLiveCaseloadDetailItem(listingDetails, submitEvent.getCaseData());
                 if (localReportsDetailItem.getValue() != null) {
                     localReportsDetailList.add(localReportsDetailItem);
                 }
             }
+
             var localReportsDetailHdr = new AdhocReportType();
             localReportsDetailHdr.setReportOffice(UtilHelper.getListingCaseTypeId(listingDetails.getCaseTypeId()));
             listingDetails.getCaseData().setLocalReportsDetailHdr(localReportsDetailHdr);
             listingDetails.getCaseData().setLocalReportsDetail(localReportsDetailList);
 
+            var localReportsSummaryHdr = new AdhocReportType();
+
+            var singlesTotal = getSinglesTotal(localReportsDetailList);
+            var multiplesTotal = getMultiplesTotal(localReportsDetailList);
+            var total = singlesTotal + multiplesTotal;
+
+            localReportsSummaryHdr.setSinglesTotal(String.valueOf(singlesTotal));
+            localReportsSummaryHdr.setMultiplesTotal(String.valueOf(multiplesTotal));
+            localReportsSummaryHdr.setTotal(String.valueOf(total));
+            listingDetails.getCaseData().setLocalReportsSummaryHdr(localReportsSummaryHdr);
         }
 
         listingDetails.getCaseData().clearReportFields();
         return listingDetails.getCaseData();
     }
 
+    private static long getSinglesTotal(List<AdhocReportTypeItem> localReportsDetailList) {
+        long singlesTotal = 0;
+        if(!localReportsDetailList.isEmpty() && localReportsDetailList.size() > 0) {
```

Comment:
There is an inconsistency here. If you're going to use the ! isEmpty approach in one place then you should use it in both
`if(!localReportsDetailList.isEmpty() && !localReportsDetailList.isEmpty()) {`